eva_submission/eload_ingestion.py

```diff
@@ -0,0 +1,116 @@
+import subprocess
+
+from ebi_eva_common_pyutils import command_utils
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.config_utils import get_pg_metadata_uri_for_eva_profile, get_mongo_uri_for_eva_profile
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query
+import psycopg2
+import pymongo
+
+from eva_submission.eload_submission import Eload
+
+
+class EloadIngestion(Eload):
+    config_section = 'ingestion'  # top-level config key
+    all_tasks = ['metadata_load', 'accession', 'variant_load']
+
+    def __init__(self, eload_number, settings_xml_file):
+        super().__init__(eload_number)
+        self.eload_cfg.set(self.config_section, 'ingestion_date', value=self.now)
+        self.settings_xml_file = settings_xml_file
```

Comment:
Why don't we get this setting file from the config as well. I know this is weird to get the config file path from another config file but it would be more convenient: `cfg['maven_settings_file']`

Reply:
sure, that's fine... I've updated.